.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+on: push
+
+name: CI
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Test
+        run: |
+          go test -v ./...
```

```suggestion
        run: go test -v
```

nits:
- is `./...` required?
- `|` tempting that multiline script fate 😉

---

🤔 Hm! robherley/go-test-action [includes](https://github.com/robherley/go-test-action/blob/main/src/runner.ts#L13) the `./...`

🤷‍♂️ Let's try!